.github/workflows/npm-publish-cht-deploy.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish cht-deploy to npm
+
+on:
+  push:
+    paths:
+      - 'scripts/deploy/**'
+
+jobs:
+  publish-cht-deploy:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./scripts/deploy
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20.x'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run tests
+        if: github.ref != 'refs/heads/master'
```

Comment:
Once we merge this check should be at the top level instead of here because the entire publish job should not run if not on master, right?